runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
> Or do you feel like this section somehow applies generically without needing to talk about a command-line API? If so, can you describe how standard streams and other file descriptors are being supplied here?

Yes, the standard streams are separate from the API. You can supply standard streams through a myriad of ways -- dup over a socket to a network connection for example. Not to mention that by creating a console at all you're duping over stdio. The point of this section IMO was to make clear that a container process can reasonably expect that fds `0`, `1` and `2` are valid file descriptors (and may be hooked up to something, but are usable for IO).